<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>eSMCs Robotino</string>
  </property>
  <property name="windowIcon">
   <iconset resource="robotino-ressources.qrc">
    <normaloff>:/robotino-icon.png</normaloff>:/robotino-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRobotino"/>
   <addaction name="separator"/>
   <addaction name="actionRobotino_Features"/>
   <addaction name="actionSMC"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionRobotino">
   <property name="icon">
    <iconset resource="robotino-ressources.qrc">
     <normaloff>:/robotino-icon.png</normaloff>:/robotino-icon.png</iconset>
   </property>
   <property name="text">
    <string>Robotino</string>
   </property>
   <property name="toolTip">
    <string>Starte Robotino</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="robotino-ressources.qrc">
     <normaloff>:/icons/task-reject.png</normaloff>:/icons/task-reject.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRobotino_Features">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="robotino-ressources.qrc">
     <normaloff>:/icons/office-chart-bar.png</normaloff>:/icons/office-chart-bar.png</iconset>
   </property>
   <property name="text">
    <string>Robotino Features</string>
   </property>
   <property name="toolTip">
    <string>Sensory feature view of Robotino</string>
   </property>
  </action>
  <action name="actionSMC">
   <property name="icon">
    <iconset resource="robotino-ressources.qrc">
     <normaloff>:/icons/SMCcircle.png</normaloff>:/icons/SMCcircle.png</iconset>
   </property>
   <property name="text">
    <string>SMC</string>
   </property>
   <property name="toolTip">
    <string>SMC</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="robotino-ressources.qrc"/>
 </resources>
 <connections/>
</ui>
